virttest/qemu_devices/qcontainer.py

```diff
@@ -2659,6 +2680,12 @@ def pcic_by_params(self, name, params, parent_bus=None):
                                 parent_bus=parent_bus,
                                 child_bus=bus)
 
+    def throttle_group_define_by_params(self, params, name):
```

Comment:
refer to images_define_by_params, we pass image_params as the argument, i.e. we get image_params by params.object_params(tag) already. so what about just passing the group_params here?

Reply:
images_define_by_params : at least have name and image_params argument, similar as this functoun, name for group name, params for the params set, i think both argument are required.

Reply:
I mean 
            def throttle_group_define_by_params(self, group_params, group_name)
we pass params.object_params(group_name) as the argument, so in the function we can:
throttle_group_parameter = group_params.get("throttle_group_parameter")

Reply:
I want to this function not only use internally. pass params instead of group_params, it have better open.
Same design can be found on memory_object_define_by_params ,dimm_device_define_by_params,input_define_by_params and so on.